### PR TITLE
Improve errors

### PR DESCRIPTION
**Pull request by adamaltman**
Use the appropriate error reference.

Augment the error description to match our current responses.

(Note: I accidentally included the package lock file... prevented in the future by adding it to the .gitignore)

### CODE REVIEW

**Comment by adamaltman** [comment]
I did not mean to include package-lock.json

File: .gitignore [hunk 1]
```diff
@@ -31,3 +31,6 @@ node_modules
 
 # Optional REPL history
 .node_repl_history
+
+# npm
+pacakge-lock.json
```
Comment (2e3s) [inline .gitignore]: looks wrong

Comment (adamaltman) [reply .gitignore]: what looks wrong about it?

Comment (2e3s) [reply .gitignore]: `pacakge` should be `package`

Comment (adamaltman) [inline .gitignore]:
```suggestion
package-lock.json
```

Comment (adamaltman) [reply .gitignore]: Thanks, I resolved it.

**Pull request merged**